mobile/iOS/starterapp/Podfile

```diff
@@ -2,7 +2,7 @@ use_frameworks!
 
 target 'OpenWhiskStarterApp' do
      platform :ios, '10.0'
-     pod 'OpenWhisk', :git => 'https://github.com/openwhisk/openwhisk-client-swift.git', :tag => '0.2.1'
+     pod 'OpenWhisk', :git => 'https://github.com/openwhisk/openwhisk-client-swift.git', :tag => '0.2.2'
```

Comment:
I'd like to keep the SDK version at 0.2.2 since it hasn't changed.  Just the starter app.